src/lib/fhirTables.js

```diff
@@ -397,3 +413,196 @@ function obsCompare(a, b) {
   return(effectiveB - effectiveA);
 }
 
+// +--------------------+
+// |    Plan of Care    |
+// +--------------------+
+
+function carePlanHeader() {
+  return (
+    <tr>
+      <th>Status</th>
+      <th>Intent</th>
+      <th>Activities</th>
+      <th>Category</th>
+      <th>Period Start</th>
+      {/* Add headers for other relevant CarePlan properties */}
```

Comment:
Also missed this last review --- I wonder if we should add the "Note" column to this? Since so much of this type is nullable (can have a valid careplan with no activities/category!!!) we might have people who just use this as a "notepad" of types? I don't know for sure.

Reply:
Great point! I have noticed those fields oddly missing from time to time - added in the Note column.